### ISSUE TITLE
Keep the same rule for the versions instead of forcing ^

### ISSUE THREAD

**Issue by lipis** [peerigon/updtr#14]
Here are some examples on how I would expect it to work out of the box:

```
~3     --> ~4
~2.1   --> ~4.2
~2.0.0 --> ~4.0.2
 1.2.0 -->  2.3.7 
 1.x.x -->  3.x.x
^2.1.9 --> ^3.0.1
 0.0.0 -->  3.0.2
```

**Comment by alexlawrence** [comment]
:+1: 

We are using exact version numbers and they need to stay exact :-)
(For our scenario it would be sufficient to use `--save-exact`)

**Comment by matthaias** [comment]
I will give it a look in the first days of 2016. I see the case of @alexlawrence, but @lipis do you really want to keep the rule of a version for minor 2 for the version with minor 4?

**Comment by matthaias** [comment]
`updtr --save-exact` is published as `v0.1.10` @alexlawrence

**Comment by alexlawrence** [comment]
Nice, thank you. Will give it a shot.
